docs/devnet.md

```diff
@@ -1,8 +1,10 @@
-# Testnet
+# Devnet
+
+Since the Chrysalis update, the `tesnet` is now known as `devnet`.  We recommend you use the `devnet` to develop and test your application.  
```

Comment:
@lucas-tortora the old testnet, before Chrysalis, was also called `devnet`, but before the upgrade on `mainnet` Luca created the network that was referenced as `testnet`. So maybe say something like `the testnet network name is devnet`?

Reply:
```suggestion
Since the Chrysalis update, the `tesnet` is now called `devnet`.  We recommend you use the `devnet` to develop and test your application.  
```